src/Ball.js

```diff
@@ -44,14 +49,44 @@ export class Ball {
       }
 
       if (this.timeInfected >= TICKS_TO_RECOVER) {
+        const oldState = this.state
         this.state = STATES.recovered
-        RUN.results[STATES.infected]--
+        RUN.results[oldState]--
```

Comment:
Puo' essere un valore diverso da infected?

Reply:
`infected` or `quarantine` (see the if block above).
Changing the ball state to quarantine doesn't affect the time you remain sick.